### PR TITLE
Update pfp and quotes upto 541

### PR DESCRIPTION
**Pull request by AterAnimAvis**
- [x] Update Names
- [x] Update Roles
- [x] Quote 490
- [x] Word Wrapping on Raven's quote (526)

### CODE REVIEW

File: quotes/data.json [hunk 1]
```diff
@@ -2096,6 +2128,476 @@
...
+            "side_text": "quoted by <strong style=\"font-size: 115%\">Raven</strong> as she fucks up a quote",
+            "text": "Illy fucked up the quote"
+        },
+        {
+            "user": "senox13",
+            "text": "What you're doing is the equivalent of trying to learn how to swim while insisting on wearing concrete shoes"
+        },
+        {
+            "user": "ravenbuilder934",
+            "text": "cue ashers with <i>'truen't'</i>"
```
Comment (ravenbuilder934) [inline quotes/data.json:2588]: But...the entire sentence was in italics, not just the `truen't`.

**Comment by AterAnimAvis** [comment]
The issue of word wrapping on 526 is left as a commented todo for the moment.

**Comment by AterAnimAvis** [comment]
Wrapping for 526 is fixed, I believe this is ready for a review, barring another pass on names/roles/pfp if necessary

**Review by sciwhiz12 — APPROVED** [review]
:shipit:

**Pull request merged**